src/main/java/mekanism/common/util/MekanismUtils.java

```diff
@@ -1493,7 +1489,7 @@ public static boolean hasUsableWrench(EntityPlayer player, BlockPos pos)
 				return true;
 			}
 			
-			if(isCoFHHammer(tool.getItem()) && ((IToolHammer)tool.getItem()).isUsable(tool, player, pos))
+			if(isCoFHHammer(tool.getItem())) // TODO Implement CoFH Hammer && ((IToolHammer)tool.getItem()).isUsable(tool, player, pos))
```

Comment:
Aaah, that package wasn't available when I had written it. I did write soft dependency, I ment optional dependency ;)

For this I'll let the COFH hammer out of this, if it's really needed it will come back in the future.